examples/text-generation/README.md

```diff
@@ -619,6 +619,93 @@ python run_generation.py \
 --flash_attention_recompute
 ```
 
+### Saving FP8 Checkpoints in Hugging Face format
+After quantizing the model, we can save it to a local path.
+
+**Note:** Before executing the command below, please refer to the "Running FP8 Models on a Single Device" section to measure the model quantization statistics.
+
+Here is an example of how to quantize and save the LLama2-7b model on a single card:
+```bash
+QUANT_CONFIG=./quantization_config/maxabs_quant_const_scales.json python run_generation.py \
+--model_name_or_path meta-llama/Llama-2-7b-hf \
+--attn_softmax_bf16 \
+--use_hpu_graphs \
+--trim_logits \
+--use_kv_cache \
+--reuse_cache \
+--use_flash_attention \
+--flash_attention_recompute \
+--bf16 \
+--batch_size 1 \
+--max_new_tokens 128 \
+--max_input_tokens 128 \
+--limit_hpu_graphs \
+--save_quantized_model_with_inc \
+--saved_model_path <model_path_on_local_disk>
+```
+
+### Loading FP8 Checkpoints saved in Hugging Face format
+
+You can load pre-quantized FP8 models using the `--load_quantized_model_with_inc` argument. The `model_name_or_path` should be the path where the model was saved locally using the previous command in "Saving FP8 Checkpoints in Hugging Face format" section.
+
+Below is an example of how to load a model with FP8 checkpoints on a single card.
+Please note that the model name is represented as `<model_path_on_local_disk>`.
+```bash
+python run_lm_eval.py \
+-o acc_load_fp8_model.txt \
+--model_name_or_path <model_path_on_local_disk> \
+--use_hpu_graphs \
+--use_kv_cache \
+--trim_logits \
+--batch_size 1 \
+--bf16 \
+--use_flash_attention \
+--flash_attention_recompute \
+--attn_softmax_bf16 \
+--bucket_size=128 \
+--bucket_internal \
+--load_quantized_model_with_inc
+```
+
+### Loading FP8 Checkpoints from Hugging Face
+You can load pre-quantized FP8 models using the `--load_quantized_model_with_inc` argument. The `model_name_or_path` should be a model name from [Neural Magic](https://huggingface.co/collections/neuralmagic/fp8-llms-for-vllm-666742ed2b78b7ac8df13127). For these models, you can use any number of cards to load them.
+
```

Comment:
please drop 1 card and keep multi-card example

Reply:
sure